inventory/group_vars/spark-rdma

```diff
@@ -31,17 +31,17 @@ keystone_user_os_config_name: alaska
 keystone_user_os_config_name_pam: alaska-pam
 keystone_user_os_project_name: p3
 
-monasca_app_name: johng-spark-rdma-test
-monasca_agent_project: p3
-monasca_agent_user: johng
-monasca_agent_keystone: !vault |
+ceph_manila_mount_monasca_app_name: johng-spark-rdma-test
```

Comment:
I guess I am OK with the old names, as they are not inside a role as such. But this is fine too.